src/fetch.js

```diff
@@ -1,24 +1,46 @@
-/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
With argument parsing and console it looks more like scripts. Maybe we should move them into `scripts` dir?